src/app/rpmostree-composeutil.cxx

```diff
@@ -198,11 +197,7 @@ rpmostree_composeutil_get_treespec (RpmOstreeContext  *ctx,
   const char *input_ref = NULL;
   if (!_rpmostree_jsonutil_object_get_optional_string_member (treedata, "ref", &input_ref, error))
     return NULL;
```

Comment:
We cna delete this stuff too now, no?

Reply:
Well spotted, those bits are recycled :recycle: now!